src/utilities/auth-utils.test.ts

```diff
@@ -23,4 +24,65 @@ describe("auth-utils", () => {
       expect(newURL?.toString()).toBeUndefined();
     });
   });
+
+  describe("getPortalStandaloneSignInOrRegisterUrl", () => {
+    // functions to mock and reset window.location and url params
+    const originalLocation = window.location;
+    const mockWindowLocation = (newLocation: Location | URL) => {
+      delete (window as any).location;
+      (window as any).location = newLocation as Location;
+      reprocessUrlParams();
+    };
+    const setLocation = (url: string) => mockWindowLocation(new URL(url));
+    afterEach(() => mockWindowLocation(originalLocation));
+
+    it("uses the authDomain param if present", () => {
+      setLocation("https://collaborative-learning.concord.org/standalone/?authDomain=https://example.com");
+      const expectedURL = "https://learn.concord.org/users/sign_in_or_register?app_name=CLUE&login_url=https%3A%2F%2Fcollaborative-learning.concord.org%2Fstandalone%2F%3FauthDomain%3Dhttps%253A%252F%252Flearn.concord.org";
+      const actualURL = getPortalStandaloneSignInOrRegisterUrl();
+      expect(actualURL).toBe(expectedURL);
+    });
```

Comment:
It appears you're specifying `example.com` for auth but the expectedURL goes to `learn.concord.org`.

Reply:
Good catch.  I changed that parameter to `portalDomain` as `authDomain` was already being used (as can be seen in the generated url).  I'll update the test in my next PR, otherwise I'll have to rebase all the chained PRs.

Reply:
Fixed in https://github.com/concord-consortium/collaborative-learning/pull/2494/commits/6f4669c63e1c6ac3f157c2f65671105e768d2024.  I decided to do it in this PR since I merged master in anyway the follow on PRs will probably need to be updated anyway.